gridded_etl_tools/utils/zarr_methods.py

```diff
@@ -412,45 +413,44 @@ def parse(self, *args, **kwargs) -> bool:
         self.info("Running parse routine")
         # adjust default dask configuration parameters as needed
         self.dask_configuration()
-        # Use a Dask client to open, process, and write the data
+        # # IPLD objects can't pickle successfully in Dask distributed schedulers so we remove the distributed client
         with LocalCluster(
             processes=self.dask_use_process_scheduler,
             dashboard_address=self.dask_dashboard_address,  # specify local IP to prevent exposing the dashboard
             protocol=self.dask_scheduler_protocol,  # otherwise Dask may default to tcp or tls protocols and choke
             threads_per_worker=self.dask_num_threads,
             n_workers=self.dask_num_workers,
-        ) as cluster, Client(
-            cluster,
-        ) as client:
-            self.info(f"Dask Dashboard for this parse can be found at {cluster.dashboard_link}")
-            try:
-                # Attempt to find an existing Zarr, using the appropriate method for the store. If there is existing data and there is no
-                # rebuild requested, start an update. If there is no existing data, start an initial parse. If rebuild is requested and there is
-                # no existing data or allow overwrite has been set, write a new Zarr, overwriting (or in the case of IPLD, not using) any existing
-                # data. If rebuild is requested and there is existing data, but allow overwrite is not set, do not start parsing and issue a warning.
-                if self.store.has_existing and not self.rebuild_requested:
-                    self.info(f"Updating existing data at {self.store}")
-                    self.update_zarr()
-                elif not self.store.has_existing or (
-                    self.rebuild_requested and self.overwrite_allowed
-                ):
-                    if not self.store.has_existing:
-                        self.info(
-                            f"No existing data found. Creating new Zarr at {self.store}."
-                        )
+        ) as cluster:
+            with Client(cluster) if not isinstance(self.store, IPLD) else nullcontext() as client:
```

Comment:
Do we want to make the nullcontext change? Or does it not matter because we're sticking with older dask

Reply:
Let’s keep it in there since it took us a while to figure out?

Reply:
Also since we’re pretty sure IPLD objects probably shouldn’t have pickled successfully in prior versions of Dask